README.md

```diff
@@ -1222,10 +1220,79 @@ seq(ap(parx, parf)).value(console.log);
 //Or concurrent sequencing
 seq(sequence(Par, [parx, parf])).value(console.log);
 //> [x, f]
+```
+
+##### Par
+
+Converts a Future to a ConcurrentFuture
+
+<details><summary><code>Par :: Future a b -> ConcurrentFuture a b</code></summary>
+
+```hs
+Par :: Future a b -> ConcurrentFuture a b
+```
+
+</details>
+
+##### Par.of
+
+Construct a ConcurrentFuture with the given resolution value.
+
+<details><summary><code>Par.of :: b -> ConcurrentFuture a b</code></summary>
+
+```hs
+Par.of :: b -> ConcurrentFuture a b
+```
+
+</details>
+
+##### Par.zero
 
-//Or racing with alternative
-seq(alt(zero(Par), parx)).value(console.log);
+Construct a ConcurrentFuture which will never resolve or reject with anything.
+
+<details><summary><code>Par.zero :: () -> ConcurrentFuture a a</code></summary>
+
+```hs
+Par.zero :: () -> ConcurrentFuture a a
+```
+
+</details>
+
+##### seq
+
+Converts a ConcurrentFuture to a Future.
+
+<details><summary><code>seq :: ConcurrentFuture a b -> Future a b</code></summary>
+
+```hs
+seq :: ConcurrentFuture a b -> Future a b
+```
+
+</details>
+
+##### alt
+
+Select one of two [Alts](#types). In terms of the `ConcurrentFuture`
+type, this means racing the two against one another with the exact same
+semantics as [`race`](#race).
```

Comment:
I would drop _exact_ from this sentence. If you'd like to keep it, I suggest _exactly the same_ in place of _the exact same_.